tests/rest/test_udp.py

```diff
@@ -21,7 +21,7 @@ def test_describe(con100, requests_mock):
 
     udp = con100.user_defined_process(user_defined_process_id='evi')
     details = udp.describe()
-
+    
```

Comment:
Can you add a new test too (I guess you wanted to do that here)

for example copy https://github.com/Open-EO/openeo-python-client/blob/bde2d0f992bd52fc244c8bfeceac4e58d6b12c2d/tests/rest/test_udp.py#L28-L51   and create a test `test_store_full` that uses and checks for all these added properties